R/getDominantTaxa.R

```diff
@@ -0,0 +1,152 @@
+#' Get Dominant Taxa
+#'
+#' This function return the most dominant taxa.
+#'
+#' @param x
+#' A \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#' object
+#'
+#' @param rank A single character defining a taxonomic rank. Must be a value of
+#'   \code{taxonomicRanks()} function.
+#'
+#' @param group With group, it is possible group observations in overview.
```

Comment:
the type of input needs to described.

Reply:
Important information... added description for group and name.